resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -224,7 +225,9 @@ cloudwatch.logs.opening_in_editor=Opening Stream {0} in editor
 cloudwatch.logs.query=Open Query Editor
 cloudwatch.logs.query.form.ok_Button=Apply
```

Comment:
"Apply" is weird, shouldn't it be "Execute"?

Reply:
Changed the text of the button to "Execute"